indigo/distribution.yaml

```diff
@@ -13174,6 +13174,21 @@ repositories:
       type: git
       url: https://github.com/srv/stereo_slam.git
       version: indigo
+  stf:
+    doc:
+      type: git
+      url: https://github.com/jstnhuang/stf.git
+      version: 0.1.0
```

Comment:
This needs to refer to a branch name. I your case `indigo-devel`.

Same for the `source` entry.